aws-cdk-eks-app-alarms-to-slack/bin/cluster-blueprint.ts

```diff
@@ -60,6 +61,7 @@ const addOns: Array<blueprints.ClusterAddOn> = [
 const stack = blueprints.EksBlueprint.builder()
```

Comment:
Could we use `ObservabilityBuilder` that we use in CDK accelerator vs the generic builder pls?